### DISCUSSION TITLE
Some possible robots

### DISCUSSION THREAD

**Discussion by fabriziosalmi** [pncsoares/youtube-video-creator-robot#76] [Ideas]
Hello, can be wondering if the team will have more people to improve the whole production process:

Maybe for the audio engineering role someone who can convert the content to the best TTS output out there will fit the accessibility requirement :)
On the video post-production workflow just look for an experienced professional who can suggest different outputs to let the boss select the best to share to the crowd!

**Comment by pncsoares** [comment]
Sounds great!
Thanks for the feedback.

**Reply by fabriziosalmi** [reply]
I wanna share my experience with You since I did a similar "team" years ago :)
I used some python tool to concatenate and go over the AWS Polly limit of the ancient TTS warriors..
Then I intensively used Wikipedia API to get content like you did, sincerily my process was weaker and prone to errors but.. who knows?
The whole content mixed by ffmpeg, adding some overlay text of course, sometimes painting waverforms (for audio files), sometimes painting a slideshow (for universe related yt channel).. in other words, you make my day if we find a time machine <3 
Keep up the good work and inflate youtube storage as much as you can ;)

**Reply by pncsoares** [reply]
Thanks a lot for the feedback @fabriziosalmi , I really appreciate it.

Unfortunately, at the moment and for the short term, I am not available to continue this project but I should be able to help with a few things.

Feel free to change what you want 😊

I also have some issues that I created in order to build a guideline and share what it's done and what needs to be done.
